fasjsonclient/clientbuilder/__init__.py

```diff
@@ -0,0 +1,20 @@
+from fasjsonclient.openapi_client import Configuration, ApiClient, DefaultApi
+from fasjsonclient.openapi_client.rest import RESTClientObject
+
+from . import patch
+
+
+class ClientBuilder(object):
+  """Class that builder a fasjson openapi rest client object"""
+  def __init__(self,
+      baseurl='http://fasjson.example.test/fasjson',
+      principal_name='admin@EXAMPLE.TEST'):
+    self.baseurl = baseurl
+    self.principal_name = principal_name
+
+  
+  def build(self):
+    patch.patch_all(principal_name=self.principal_name)
+    config = Configuration(host=self.baseurl)
+    api_client = ApiClient(config)
+    return DefaultApi(api_client)
```

Comment:
This could probably be a function rather than a class, no?

Reply:
I used a class so it can be easily changed/configurable

Reply:
At the moment it doesn't seem better than a constructor function that would take arguments, I'd rather start simple and move to a class that has a `__call__` attribute if necessary. But factories are frequently functions.